Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM python:3.8 as base
+
+# Install Poetry
+RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+ENV PATH /root/.poetry/bin:$PATH
+
```

Comment:
You might want to set some other environment variables.

E.g. you could set `ENV FLASK_APP=todo_app/app`

And you could set `ENV FLASK_ENV=production` inside the production stage.